### PR TITLE
Style checkout

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: woocommerce/checkout/form-shipping.php [hunk 1]
```diff
@@ -21,24 +21,26 @@
 }
 
 ?>
-<div class="woocommerce-shipping-fields grid-column">
+<div class="woocommerce-shipping-fields">
 	<?php if ( true === WC()->cart->needs_shipping_address() ) : ?>
 
-		<div class="shipping_address grid-column">
+		<div class="shipping_address grid-row margin-top-small padding-top-basic">
```
Comment (varffs) [inline woocommerce/checkout/form-shipping.php:27]: tab indentation 👎

**Comment by varffs** [comment]
Look good except for the notices issue again. I wonder if better to just do another branch putting the notices in a styled box on checkout, cart and shop single pages?

screen shot 2016-12-20 at 18 21 42

File: woocommerce/checkout/form-shipping.php (hunk 1 above)
Comment (mikeyrayvon) [reply woocommerce/checkout/form-shipping.php:27]: its woocommerce templates dude

Comment (varffs) [reply woocommerce/checkout/form-shipping.php:27]: you can fix it tho with a search replace or probably some IDE plugin

**Comment by mikeyrayvon** [comment]
no they should be removed.

**Pull request merged**